scripts/system_performance/system_performance_queries.py

```diff
@@ -0,0 +1,156 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+
+"""
+The purpose of this script is for performing MySQL queries to monitor system state
+performance and health.
+"""
+from __future__ import print_function
+import itertools
+import logging
+from utils.clients.database_client import DatabaseClient
+
+
+class DatabaseMonitorChecks(object):
+    """Class to query system performance"""
+    table = "reduction_viewer_reductionrun"
+
+    # Establishing a connection with Database using Database Client
+    def __init__(self):
+        self.database = DatabaseClient()
+        self.connection = self.database.connect()
+
+    def query_log_and_execute(self, constructed_query):
+        """Logs and executes all queries ran in script"""
+        logging.info('SQL QUERY: %s', constructed_query)
+        print(constructed_query)
+        return self.connection.execute(constructed_query).fetchall()
+
+    def get_instruments_from_database(self):
+        """Retrieve current list of instruments from database"""
+        all_instruments = "SELECT id, name "\
+                          "FROM reduction_viewer_instrument"
+        return self.query_log_and_execute(all_instruments)
+
+    def rb_range_by_instrument(self, instrument, start_date, end_date):
+        """Retrieves run_numbers as longs for a given instrument
```

Comment:
Doctsring @params to specify input types for start/end date and needs an @return docstring too